### PR TITLE
polsedit: Add version 1.0.0.5

### PR DESCRIPTION
**Pull request by TamamoNoMae13**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/polsedit.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "latest",
```
Comment (rashil2000) [inline bucket/polsedit.json:2]:
```suggestion
    "version": "1.0.0.5",
```

File: bucket/polsedit.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.southsoftware.com/terms.html"
+    },
+    "notes": "This application requires Administrator privileges to run.",
+    "url": "https://www.southsoftware.com/polsedit.zip",
+    "hash": "149c3578103f43482f013238cd4fc0db354a862d9acffdbe9c2c2f0e80a8f794",
+    "architecture": {
+        "64bit": {
+            "bin": "polseditx64.exe",
```
Comment (rashil2000) [inline bucket/polsedit.json:14]:
```suggestion
            "bin": [
                [
                    "polseditx64.exe",
                    "polsedit"
                ]
            ],
```

File: bucket/polsedit.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            "shortcuts": [
+                [
+                    "polseditx64.exe",
+                    "Polsedit - User Policies Editor"
+                ]
+            ]
+        },
+        "32bit": {
+            "bin": "polseditx32.exe",
```
Comment (rashil2000) [inline bucket/polsedit.json:23]:
```suggestion
            "bin": [
                [
                    "polseditx32.exe",
                    "polsedit"
                ]
            ],
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
This is a GUI application, right? Does it support CLI arguments?

**Comment by TamamoNoMae13** [comment]
> This is a GUI application, right? Does it support CLI arguments?

As far as I tried, no.

**Comment by rashil2000** [comment]
In that case the bin fields can be removed

**Pull request merged**